tests/drivers/dma/loop_transfer/src/test_dma_loop.c

```diff
@@ -1,17 +1,30 @@
 /* dma.c - DMA test source file */
 
 /*
- * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2021 Linaro Limited.
```

Comment:
You should not be replacing copyrights, just add yours on top.

Reply:
Ho, right, sorry, I did it mechanically as this appeared to be a new file (?!!?), will fix.

Reply:
fixed